source/stylesheets/_settings.scss

```diff
@@ -1,18 +1,58 @@
 // Breakpoints
-$desktop: new-breakpoint(min-width 768px 12);
-$tablet: new-breakpoint(max-width 767px);
-$mobile: new-breakpoint(max-width 480px);
+$desktop: new-breakpoint(min-width 800px 12);
+$tablet: new-breakpoint(max-width 799px);
+$mobile: new-breakpoint(max-width 520px);
+
 
 // Colors
 $yellow-tribune: #ffc200;
 $teal-tribune: #008990;
 
+$black: #000000;
+$gray-medium: #aaaaaa;
+$gray-light: #ebebeb;
+$white: #ffffff;
+
 $bg-contribute: white;
 $bg-nav: #000000;
 
 $link-nav: white;
 
-// Typography
+
+// Fonts
+$league-gothic: 'league-gothic', sans-serif;
+
 
 // Mixins
+@mixin hide-for-mobile {
+  @include media($mobile) {
+    display: none;
```

Comment:
Small point: for accessibility reasons, you might consider using `visually-hidden` instead. But this assumes content parity between all devices, so that may or may not apply, and it may or may not work w/ the js. Just a thought. Cf.  https://css-tricks.com/places-its-tempting-to-use-display-none-but-dont/